assets/source/setup-guide/app/steps/ClaimWebsite.js

```diff
@@ -43,11 +44,20 @@ const StaticError = ( { reqError } ) => {
 	);
 };
 
+/**
+ * Clicking on an external documentation link.
```

Comment:
Seems like a missed docs.

Reply:
Could you elaborate?

Reply:
Fixed here https://github.com/woocommerce/pinterest-for-woocommerce/pull/300/commits/7ea85eacea5235392c76c3080405733468aea7c5